src/main/scala/viper/gobra/frontend/info/base/SymbolTable.scala

```diff
@@ -245,4 +238,12 @@ object SymbolTable extends Environments[Entity] {
   case class BuiltInFPredicate(tag: BuiltInFPredicateTag, rep: PNode, context: ExternalTypeInfo) extends BuiltInGhostEntity
   case class BuiltInMPredicate(tag: BuiltInMPredicateTag, rep: PNode, context: ExternalTypeInfo) extends BuiltInGhostEntity with BuiltInMethodLike with GhostTypeMember
 
+
+
+
+  /**
+    * Label
+    */
+
+  case class Label(decl: PLabeledStmt, ghost: Boolean) extends Entity with Product
```

Comment:
What's the purpose of having a `ghost` flag here? Shouldn't a Label be always non-ghost, even if the labeled statement is ghost?

Reply:
I think I see why now. Is it because a label may occur inside a ghost function or block, in which case you want to ensure that regular control flow does not jump to a ghost statement?

Reply:
yes